src/components/Login/Login.tsx

```diff
@@ -63,33 +74,50 @@ const Login = () => {
               <LogoPage></LogoPage>
               <Label htmlFor="email">
                 <IconText />
-                email
+                {t("user.email.name")}
               </Label>
               <Input
                 type="email"
                 name="email"
                 autoCapitalize="off"
                 autoCorrect="off"
                 autoComplete="email"
-                placeholder="Type your email"
+                placeholder={t("user.email.placeholder")}
               />
               <ErrorMessage name="email">
                 {(msg) => <StyledInlineErrorMessage>{msg}</StyledInlineErrorMessage>}
               </ErrorMessage>
               <Label htmlFor="password">
                 <IconPassword />
-                password
+                {t("user.password.name")}
               </Label>
-              <Input type="password" name="password" placeholder="Type your password" />
+              <View>
+                <Input
+                  type={passwordShown ? "text" : "password"}
+                  name="password"
+                  placeholder={t("user.password.placeholder")}
+                />
+                {formValue.values.password.length > 0 ? (
```

Comment:
myślę, że tutaj może się sprawdzić operator '&&' zamiast '?'

Reply:
Próbowałam i jak zrobiłam  
```
{formValue.values.password.length > 0 && passwordShown ? 
                     <IconEye onClick={togglePassword} /> : <IconEyeHide onClick={togglePassword} /> } 

```
to miałam taki efekt:
<img width="497" alt="Screen Shot 2022-04-08 at 2 39 59 PM" src="https://user-images.githubusercontent.com/30056795/162437368-9b3ca817-545a-4efa-8b40-a59445ca4a90.png">

a mi zależało żeby ta ikonka się nie pokazywała jak jest puste. Chyba że źle ją zapisałam